src/main/java/io/github/perrymant/moneymaker/AccountRepository.java

```diff
@@ -0,0 +1,37 @@
+package io.github.perrymant.moneymaker;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+
+
```

Comment:
Please reformat this class. Or better yet, reformat the entire project. You should be able to just select the `moneymaker` package and choose reformat.